kyuubi-ha/src/main/scala/org/apache/kyuubi/ha/client/DiscoveryClient.scala

```diff
@@ -164,6 +169,13 @@ trait DiscoveryClient extends Logging {
       basePath: String,
       initData: String,
       useProtection: Boolean = false): Unit
+
+  /**
+   * Atomically get an Int number and add one
+   * @param path the path of stored data
+   * @return the stored data under path
+   */
+  def getAndInc(path: String): Int
```

Comment:
Sorry for the previously immature suggestion. I checked the JDK API, `AtomicInteger` and `AtomicLong` name the method `getAndIncrement`, do you mind updating to align w/ them? And please mention the cornet behavior in the doc, e.g. what's the behavior if the path does not exist?

Reply:
done